### PR TITLE
Add Waifudex, Urcdkey, AMD rewards, Voicemod & DLive

### PR DESCRIPTION
**Pull request by mattheroit**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -4931,6 +4931,16 @@
         ]
     },
 
+    {
+        "name": "DLive",
+        "url": "https://dlive.tv/s/settings",
+        "difficulty": "easy",
+        "notes": "In settings scroll to the bottom, click the delete account button and enter your password. Your account will be deleted in 30 days.",
+        "domains": [
+            "dlive.tv"
```
Comment (Kefaku) [inline _data/sites.json:4940]:
```suggestion
            "dlive.tv",
            "community.dlive.tv",
            "help.dlive.tv"
```

**Review by Kefaku — CHANGES_REQUESTED** [review]
There are more domains for DLive

File: _data/sites.json [hunk 2]
```diff
@@ -1052,6 +1052,16 @@
         ]
     },
 
+    {
+        "name": "AMD Rewards",
+        "url": "https://www.amdrewards.com/my-account",
+        "difficulty": "impossible",
+        "notes": "You can not delete your account, only dissable it, At the bottom of the page is a 'Deactivate your account' ⮕ Enter password ⮕ Deactivate account",
```
Comment (tupaschoal) [inline _data/sites.json:1059]:
```suggestion
        "notes": "You can not delete your account, only disable it, At the bottom of the page there is a 'Deactivate your account' ⮕ Enter password ⮕ Deactivate account",
```

File: _data/sites.json [hunk 3]
```diff
@@ -18936,6 +18958,16 @@
...
+    {
+        "name": "Urcdkey",
+        "url": "https://www.urcdkey.com/",
+        "difficulty": "hard",
+        "notes": "Send a message using the website chat feature (bottom right), asking them to delete your account",
+        "domains": [
+            "urcdkey.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:18970]:
```suggestion

```

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**